### DISCUSSION TITLE
Fs2 telescope control

### DISCUSSION THREAD

**Discussion by PaoloSpeggiorin** [Stellarium/stellarium#1923] [General]
I find problems in driving my telescope with FS2 controller and Stellarium.
I chose Direct Stellarium Control type, but I’m able only to see on the screen the pointer showing where the telescope is directed,after the star alignment; I see the pointer following the scope when I drive it with the hand-controller, but I’m not able to drive it by the keyboard of my PC. (I have Stellarium 0.21.1 version installed). Any ideas? Thank you!

**Comment by gzotti** [comment]
Duplicate of #1329 ?

**Reply by PaoloSpeggiorin** [reply]
No, I don’t have a problem of wrong coordinates reading, but the fact that The control is only “passive”, not “active”: I’m not able to drive the telescope from PC keyboard; I’m only able to see in Stellarium screen where is pointed the telescope.
